encoder/vaapiencoder_base.cpp

```diff
@@ -568,6 +568,7 @@ YamiStatus VaapiEncoderBase::getOutput(VideoEncOutputBuffer* outBuffer, bool wit
     if (ret != YAMI_SUCCESS)
         return ret;
 
+    outBuffer->timeStamp = picture->m_timeStamp;
```

Comment:
Maybe better to just do this inside the picture->getOutput(outBuffer) call instead?